File: model/src/main/java/com/nigeleke/cribbage/model/Suit.java
package com.nigeleke.cribbage.model;

public enum Suit {
    Clubs("\u2663"),
    Diamonds("\u2666"),
    Hearts("\u2665"),
    Spades("\u2660");

    public final String smallSuit;

    private Suit(String smallSuit) {
        this.smallSuit = smallSuit;
    }

}

File: packages/domain/src/card/suit.rs
use strum::EnumIter;

/// The four suits in a standard French playing card deck.
#[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash, EnumIter)]
#[rustfmt::skip]
pub enum Suit {
    #[doc(hidden)] Hearts,
    #[doc(hidden)] Clubs,
    #[doc(hidden)] Diamonds,
    #[doc(hidden)] Spades,
}

impl std::fmt::Debug for Suit {
    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
        f.write_str(match self {
            Suit::Hearts => "H",
            Suit::Clubs => "C",
            Suit::Diamonds => "D",
            Suit::Spades => "S",
        })
    }
}
